src/applications/combined-debt-portal/combined/containers/CombinedStatements.jsx

```diff
@@ -283,156 +283,151 @@ const CombinedStatements = () => {
-          <p className="vads-u-margin-top--0">
-            You are receiving this billing statement because you are currently
-            enrolled in a priority group requiring copayments for treatment of
-            non-service connected conditions.
-          </p>
-
-          <div className="vads-u-margin-top--3">
-            <h3 className="vads-u-font-size--h3 vads-u-margin-bottom--1">
-              Resolve your copay bills
-            </h3>
-            <p className="vads-u-margin-top--0">
-              You can pay your debt online, by phone, or by mail. Call us at{' '}
-              <va-telephone contact="8664001238" /> to discuss payment options,
-              request financial help, or dispute your bill.
-            </p>
-            <va-link-action
-              href="/manage-va-debt/summary/copay-balances"
-              text="Review and resolve copay bills"
-              type="secondary"
-              data-testid="review-copays-link"
-            />
-          </div>
-
-          {/* Copay charges tables */}
-          {mcp.statements.map(statement => (
-            <div key={statement.station.facilityName}>
-              <h3>{statement.station.facilityName}</h3>
-              <p className="vads-u-margin-bottom--0">
-                Payments made after{' '}
-                {getLatestPaymentDateFromCopayForFacility(statement)} will not
-                be reflected here
-              </p>
-              <va-table
-                table-title={`Copay charges for ${
-                  statement.station.facilityName
-                }`}
-                data-testid={`combined-statements-copay-table-${
-                  statement.station.facilityName
-                }`}
-                className="vads-u-width--full"
-              >
-                <va-table-row slot="headers">
-                  <span>Description</span>
-                  <span>Billing reference</span>
-                  <span>Amount</span>
-                </va-table-row>
-
-                {statement.details.map((charge, idx) => (
-                  <va-table-row key={idx}>
-                    <span>{cleanHtmlEntities(charge.pDTransDescOutput)}</span>
-                    <span>{charge.pDRefNo}</span>
-                    <span>{currency(charge.pDTransAmt)}</span>
-                  </va-table-row>
-                ))}
-
-                {copayTotalRow(statement)}
-              </va-table>
-            </div>
-          ))}
-        </section>
-
-        <section className="vads-u-margin-y--2 vads-u-padding-y--2 vads-u-border-color--gray-light ">
-          <h2 className="vads-u-margin-top--0 vads-u-margin-bottom--2">
-            Overpayment charges
-          </h2>
-          <p className="vads-u-margin-top--0">
-            Benefit overpayments are due to changes in your benefits which
-            result in you being paid more than you were owed.
-          </p>
+        <h2 className="vads-u-margin-top--0 vads-u-margin-bottom--2">
```

Comment:
I think we might want to replace the sections with `<div className="vads-u-margin-y--4 vads-u-padding-y--2"`> to maintain the OG spacing and semantic grouping. Unless you saw a reason to remove the classes or block around the content?